.gitignore

```diff
@@ -11,3 +11,6 @@ packaging/unattended_setup.json
 .ycm_extra_conf.py
 compile_commands.json
 CMakeLists.txt.user
+
+# old submodule
+/irods_schema_messaging
```

Comment:
Can the use of a leading forward slash be misinterpreted by `git`?

Reply:
seems okay?

https://stackoverflow.com/a/38433961

Reply:
The leading forward slash matches the root of the tree, which is what we want.